### DISCUSSION TITLE
Add some toturial about how to write parser in rowan

### DISCUSSION THREAD

**Discussion by IWANABETHATGUY** [rome/tools#2370] [Suggestions]
### Description

Currently, rome only support Javascript and Typescript, but it seems that there still a lot of language need to support, More over, frontend actually have a lot of DSL like scss, less and so on. I think a toturial about json parser is fairly enough.

**Comment by IWANABETHATGUY** [comment]
Since rowan repo has serval sample about how to write simple parser, it would be better focus on how to combine ungrammar and rowan, error recovery

**Comment by ematipico** [comment]
Yeah this is something we had in mind for long time, but haven't had time to commit, mostly because so far our parser infrastructure continued to change until not long time ago. 

Soon we'll start creating new parsers (from scratch or using existing tools), so yeah it might be a good opportunity to create some blog post.

There's one thing to note though, our fork of rowan has changed a lot, so the tutorials won't use the original crate.
